pom.xml

```diff
@@ -61,6 +61,11 @@
       <version>1.3</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+      <artifactId>icon-shim</artifactId>
```

Comment:
No opinion - the plugin has been designed for this version, docker-traceability supports it as well